tasks/main.yaml

```diff
@@ -0,0 +1,77 @@
+---
+- name: Install required pakages
+  package:
+    name: "{{ item }}"
+    state: present
+  with_items: "{{ capra_lvm_filesystem_packages }}"
+
+- name: "Configure default mountpoint directory"
+  file:
+    path: "{{ capra_lvm_filesystem_mount_parent }}"
+    state: directory
+    owner: "{{ capra_lvm_filesystem_owner }}"
+    group: "{{ capra_lvm_filesystem_group }}"
+    mode: 0755
+    seuser: unconfined_u
+    serole: object_r
+    setype: default_t
+    selevel: s0
+
+- name: "Unmount volume"
+  mount:
+    path: "{{ item.mount | default(capra_lvm_filesystem_mount_parent + '/' + item.name) }}"
```

Comment:
Use `~` to concatenate values. `mount` can mean many things, hence other places use `mountpath`. Also, why duplicate this logic multiple times instead of defining a module-global variable making use of `item`? Now that it's mentioned: Rename the loop variable with a prefix.

Reply:
There is still one instance of duplication left in the task using `with_indexed_items`. Any suggestions to get rid of that one?

Reply:
Found a solution for that one.